lib/commands/submission.js

```diff
@@ -96,6 +96,10 @@ function exportSubmission(argv, problem, cb) {
         submissionState,
         h.langToExt(submission.lang));
 
+    if (!fs.existsSync(argv.outdir)) {
+      fs.mkdirSync(argv.outdir);
```

Comment:
unfortunately this won't create directories recursively, while I think it's good enough for now.

Reply:
If you want that, I can implement it but I think there is no need for it.